packages/block-editor/src/hooks/align.js

```diff
@@ -94,6 +94,9 @@ export function addAttribute( settings ) {
 			...settings.attributes,
 			align: {
 				type: 'string',
+				// Allow for '' since it is used by updateAlignment function
+				// in withToolbarControls for special cases with defined default values.
+				enum: [ ...ALL_ALIGNMENTS, '' ],
```

Comment:
Updated this - It seems we need to allow the empty string as a value as it is used for special cases when default values are defined:
https://github.com/WordPress/gutenberg/blob/74c9065935fe995fba06f9c21ef9fb09f06dd6b2/packages/block-editor/src/hooks/align.js#L140-L142